### ISSUE TITLE
File not updated.

### ISSUE THREAD

**Issue by JohnHunsberger** [JohnHunsberger/JohnHunsberger.github.io#1]
Two days ago I corrected a link in master/HTML_unscripted/unscripted-AboutThisPage.html file.  The change shows correctly when I view the file source on GitHub.  I used GitHub Desktop to commit the change to the master.  The change is not showing up as being published at the site johnhunsberger.com.  I tried both chrome and firefox.  I cleared the browser history cache several times so it isn't a local browser problem.  Can you offer advice on how to get around this issue?  I appreciate your time to respond.  Thanks.

**Comment by JohnHunsberger** [comment]
Hello.  I finally see this file updated.  Not certain why however  yesterday I made two separate minor changes to the file, one on GitHub with a post and one using Atom with a post in GitHub Desktop.  This morning I checked johnhunsberge.com and saw that the file did update.  Anyway, thanks for being there to support users.  Some stuff for me could just be ignorance and I sincerely do not want to waste anybody's time with new-be issues.  Regards,
